examples/gallery/plot/solar.py

```diff
@@ -0,0 +1,17 @@
+"""
```

Comment:
The gallery is good, but I like this example https://docs.generic-mapping-tools.org/dev/gallery/ex46.html more.

Do you mind rewrite this example (you don't have to plot the sun)?